test/test_datasets.py

```diff
@@ -72,13 +72,24 @@ def test_openfire(self):
             self.assertEqual(dataset.class_to_idx[extract[0]['target']], target)
 
             # Check train/test split
-            train_set = datasets.OpenFire(root=root, train=True, download=True, valid_pct=0.2)
-            test_set = datasets.OpenFire(root=root, train=False, download=True, valid_pct=0.2)
+            train_set = datasets.OpenFire(root=root, train=True, download=True, valid_ratio=0.2)
+            test_set = datasets.OpenFire(root=root, train=False, download=True, valid_ratio=0.2)
             self.assertIsInstance(train_set, VisionDataset)
             # Check unicity of sample across all splits
             train_paths = [sample['path'] for sample in train_set.data]
             self.assertTrue(all(sample['path'] not in train_paths for sample in test_set.data))
 
+    def test_openfire_validation(self):
+        """Check if Exception is raised when providing incorrect split ratio for validation"""
+
+        with Path(tempfile.TemporaryDirectory().name) as root:
+
+            incorrect_valid_ratios = [-25, -0.01, 1.01, 15]
```

Comment:
It might be safer to add non-numerical values to test the behaviour from the previous comment.